src/map/ai/ai_mob_dummy.cpp

```diff
@@ -719,6 +719,7 @@ void CAIMobDummy::ActionAbilityStart()
     // We don't have any skills we can use, so let's go back to attacking
     if (MobSkills.size() == 0)
     {
+        m_PMob->PBattleAI->SetMobAbilityEnabled(false);
         TransitionBack(true);

```

Comment:
This would break mob's that don't have any skills and then later on get skills. I think the best solution is to prevent the mob from going into this method if it doesn't have any skills. Add the check to line 1427 and 1905.